apps/trustlab/src/components/RichText/RichText.js

```diff
@@ -0,0 +1,25 @@
+import { Box } from "@mui/material";
+import { RichText as ConvertRichText } from "@payloadcms/richtext-lexical/react";
+import { forwardRef } from "react";
+
+import { styleConverter } from "./converter";
+
+export const jsxConverters =
+  (converterProps) =>
+  ({ defaultConverters }) => ({
+    ...defaultConverters,
+    ...styleConverter(converterProps),
+  });
+
+const RichText = forwardRef(function RichText(props, ref) {
```

Comment:
Shouldn't this be in `@commons-ui/payload` package? I assume it's the same component used in `roboshield` for example.